mteb/cli.py

```diff
@@ -105,7 +106,12 @@ def run(args: argparse.Namespace) -> None:
 
     logger.info("Running with parameters: %s", args)
 
-    model = mteb.get_model(args.model, args.model_revision, device=args.device)
+    if args.device is None:
+        device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
Should we generally use `"cuda"` or `torch.device("cuda")`? We use the latter in the e5 model code & the former here. Don't have a preference, maybe one or the other is better?

Reply:
Either will work for the e5, but the interface specified a string. Either is fine with me